deployments/warp_routes/README.md

```diff
@@ -0,0 +1,15 @@
+Directory structure is as follows:
+
+```
+<ASSET_SYMBOL>/<ORIGIN_CHAIN>/<DESTINATION_CHAIN>/addresses.json
+```
+
+Addresses are keyed by their token router type:
+
+```typescript
+enum TokenType {
+  synthetic,
+  collateral,
+  native,
+}
+```
```

Comment:
also if it's possible, maybe we can directly read from the monorepo similar to the docs